django_select2/conf.py

```diff
@@ -34,6 +42,12 @@ class Select2Conf(AppConf):
     .. tip:: To ensure a consistent state across all you machines you need to user
         a consistent external cache backend like Memcached, Redis or a database.
 
+    .. note:: If you are having problems with the fetching of the results
```

Comment:
I have a working example with memcached. It is supported. Please drop that comment. This could be missleading for some.

Reply:
I've made change in the last commit, I think it was the case too keep the thing about redis-server (that solved my problem after all, that i had not the package installed)